cvmfs/swissknife_check.cc

```diff
@@ -35,8 +35,24 @@
 
 using namespace std;  // NOLINT
 
+// for map of duplicate entries; as in kvstore.cc
+static inline uint32_t hasher_any(const shash::Any &key) {
+  // We'll just do the same thing as hasher_md5, since every hash is at
+  // least as large.
+  return *const_cast<uint32_t *>(
+             reinterpret_cast<const uint32_t *>(key.digest) + 1);
+}
+
+
 namespace swissknife {
 
+CommandCheck::CommandCheck()
+    : check_chunks_(false)
+    , is_remote_(false) {
+    const shash::Any hash_null(shash::kMd5);
```

Comment:
In principle correct but unclear why we bother picking the MD5 algorithm. Let's just do
```suggestion
    const shash::Any hash_null;
```